### PR TITLE
Added test for handleChangeRef in useForm

### PR DESCRIPTION
**Pull request by keiya01**
- I fixed test for `register`.
- I added test for `handleChangeRef`.

### CODE REVIEW

File: src/useForm.test.tsx [hunk 1]
```diff
@@ -1593,4 +1633,395 @@ describe('useForm', () => {
...
+      validFields: new Set(),
+      fieldsWithValidation: new Set(),
+    };
+    let renderCount: number;
+    let Component: React.FC<{
+      resolver?: any;
+      mode?: 'onBlur' | 'onSubmit' | 'onChange';
+    }>;
+    let methods: UseFormMethods<{ test: string }>;
+    beforeEach(() => {
```
Comment (bluebill1049) [inline src/useForm.test.tsx:1678]:
```suggestion

    beforeEach(() => {
```

File: src/useForm.test.tsx [hunk 2]
```diff
@@ -1593,4 +1633,395 @@ describe('useForm', () => {
...
+        mode?: 'onBlur' | 'onSubmit' | 'onChange';
+      }) => {
+        const internationalMethods = useForm<{ test: string }>({
+          resolver,
+          mode,
+        });
+        const { register, handleSubmit, errors } = internationalMethods;
+        methods = internationalMethods;
+        renderCount++;
+        return (
```
Comment (bluebill1049) [inline src/useForm.test.tsx:1706]:
```suggestion

        return (
```

**Review by bluebill1049 — DISMISSED** [review]
Awesome stuff!! thank you again @keiya01 ❤️

**Comment by keiya01** [comment]
Thank you for reviewing!!

**Pull request merged**